docs/source/compute_config/aws_lambda.md

```diff
@@ -52,26 +51,69 @@ python3 -m pip install lithops[aws]
 lithops:
     backend: aws_lambda
 
-aws:
+aws_lambda:
+    execution_role: <EXECUTION_ROLE_ARN>
     region: <REGION_NAME>
+```
+
+## AWS Credential setup
+
+Lithops loads AWS credentials as specified in the [boto3 configuration guide](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html).
+
+In summary, you can use the following settings:
+
+1. Provide credentials via the `~/.aws/config` file. **This is the preferred option to configure AWS credentials for use with Lithops**:
+
+    You can run `aws configure` command if the AWS CLI is installed to setup the credentials.
+
+2. Provide credentials via environment variables:
+
+    Lithops needs at least `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` and `AWS_DEFAULT_REGION` environment variables set.
+
+3. Provide the credentials in the `aws` section of the Lithops config file **This option is not ideal and will be removed in future Lithops releases!**:
```

Comment:
No need for any warning for now. We still have to decide whether we want to deprecate it or not